laravel/app/Http/Controllers/OrderController.php

```diff
@@ -62,4 +63,23 @@ public function order_delete_execute(Request $request)
         session()->flash('flash_message', '注文の消去完了しました');
         return redirect('/orders');
     }
+
+    public function auto_delivery_manual_execute(Request $request)
```

Comment:
manualということはautoではないので、下記の様な感じでも良いかもしれません
```suggestion
    public function manual_delivery__execute(Request $request)
```